components/omega/src/ocn/EosConstants.h

```diff
@@ -0,0 +1,92 @@
+#ifndef OMEGA_EOSCONSTANTS_H
+#define OMEGA_EOSCONSTANTS_H
+//===-- ocn/Eos.h - Equation of State --------------------*- C++ -*-===//
+//
+/// \file
+/// \brief Contains functors for calculating specific volume
+///
+/// This header defines functors to be called by the time-stepping scheme
+/// to calculate the specific volume based on the choice of EOS
+//
+//===----------------------------------------------------------------------===//
+
+namespace OMEGA {
+
+#define GSW_SPECVOL_COEFFICIENTS
```

Comment:
I think I prefer to actually inline these in the TEOS function rather than having the separate header and DEF.  But if we decide to keep this, we don't need the namespace qualifier here since they are in a single function scope anyway.